drivers/SmartThings/matter-switch/src/test/test_matter_switch_device_types.lua

```diff
@@ -67,7 +67,7 @@ local mock_device_dimmer = test.mock_device.build_test_matter_device({
   },
   endpoints = {
     {
-      endpoint_id = 5,
+      endpoint_id = 0,
```

Comment:
Why is this changed to endpoint 0? Endpoint 5 is a valid starting endpoint

Reply:
This was changed while there were issues in the code previously and I forgot to restore it. Fixed!